src/python/grpcio/grpc/experimental/aio/__init__.py

```diff
@@ -25,10 +25,12 @@
 from ._call import Call
 from ._channel import Channel
 from ._channel import UnaryUnaryMultiCallable
+from ._interceptor import ClientCallDetails
+from ._interceptor import UnaryUnaryClientInterceptor
 from ._server import server
 
 
-def insecure_channel(target, options=None, compression=None):
+def insecure_channel(target, options=None, compression=None, interceptors=None):
```

Comment:
We may also want to add type annotations to these function signatures. `__init__.py` is where they'll be most useful to users.